crates/bifrost/src/bifrost.rs

```diff
@@ -92,6 +93,13 @@ impl Bifrost {
     ///
     /// It's recommended to use the [`Appender`] interface. Use [`Self::create_appender`]
     /// and reuse this appender to create a sequential write stream to the virtual log.
+    #[instrument(
+        level="trace",
+        skip(self, body),
```

Comment:
nit: `skip_all` and then choosing what to include might be a tad bit more robust against signature changes.

Reply:
There is actually also the `log_id` in the argument list which i wanted to keep :)

Reply:
Yeah, I guess this would then translate to `skip_all, fields(%log_id)`. This is not important, though.